docs_outdoornav_user_manual/api/api_endpoints.mdx

```diff
@@ -557,13 +557,238 @@ heading.
 **Description:** Send the robot to the charging dock, if the docking
 package has been included.
 
+------------------------------------------------------------------------
+
+## Mission API Services {#services-mission-api}
+
+The Mission Manager API allows creation, deletion, and modification of OutdoorNav missions.  Mission data is stored

```

Comment:
Just for clarity, is it the "Mission API" or the "Mission Manager API"? Either one is probably ok, but we should be consistent throughout. Probably "Mission Manager API" since the user can't actually run missions through this API, which one might expect to be able to do with a "Mission API".